docs/servicenow.itsm.incident_info_module.rst

```diff
@@ -353,7 +353,7 @@ Parameters
                 </td>
                 <td>
                         <div>Provides a set of operators for use with filters, condition builders, and encoded queries.</div>
-                        <div>The data type of a field determines what operators are available for it. Refer to the ServiceNow Available Filters Queries documentation at <a href='https://docs.servicenow.com/bundle/quebec-platform-user-interface/page/use/common-ui-elements/reference/r_OpAvailableFiltersQueries.html'>https://docs.servicenow.com/bundle/quebec-platform-user-interface/page/use/common-ui-elements/reference/r_OpAvailableFiltersQueries.html</a>.</div>
+                        <div>The data type of a field determines what operators are available for it. Refer to the ServiceNow Available Filters Queries documentation at <a href='https://docs.servicenow.com/bundle/vancouver-platform-user-interface/page/use/common-ui-elements/reference/r_OpAvailableFiltersQueries.html'>https://docs.servicenow.com/bundle/vancouver-platform-user-interface/page/use/common-ui-elements/reference/r_OpAvailableFiltersQueries.html</a>.</div>
```

Comment:
@gscarbor Thanks for the contribution. RST files are generated at the time of release, please update the respective module files (in this `modules/incident_info.py`.